Server/VikingVault.Services/LoginService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using VikingVault.DataAccess;
+using VikingVault.DataAccess.Models;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class LoginService : ILoginService
+    {
+        private VikingVaultDbContext _context;
+
+        private readonly AppSettings _appSettings;
+
+        public LoginService(VikingVaultDbContext context, IOptions<AppSettings> appSettings)
+        {
+            _context = context;
+            _appSettings = appSettings.Value;
+        }
+
+        public User Authenticate(string email, string password)
+            {
+                var user = _context.User.SingleOrDefault(x => x.Email == email && x.Password == password);
```

Comment:
It's better to use a more suggestive argument name for lambda, instead of "x" you could use "u" (as the entity is the User) `.SingleOrDefault(u => uEmail == email && u.Password == password);`